ferris/plugins/commands/converters.py

```diff
@@ -1,10 +1,17 @@
-from ferris.errors import NotFound
```

Comment:
And you need `from __future__ import `annotation` in order to use if `TYPE_CHECKING` guard